src/util/interpolate.ts

```diff
@@ -37,6 +39,10 @@ function number(from: number, to: number, t: number): number {
     return from + t * (to - from);
 }
 
+function projection(from: string, to: string, interpolation: number): ProjectionDefinition {
```

Comment:
Other method names math the type, consider changing this method badge as well.

Reply:
i've remove the function, we're fine with the default passthrough case here:

```ts
        switch (this.operator) {
            case 'interpolate':
                switch (this.type.kind) {
                    case 'number':
                        return interpolate.number(outputLower, outputUpper, t);
                    case 'color':
                        return interpolate.color(outputLower, outputUpper, t, 'rgb');
                    case 'array':
                        return interpolate.array(outputLower, outputUpper, t);
                    case 'padding':
                        return interpolate.padding(outputLower, outputUpper, t);
                    case 'variableAnchorOffsetCollection':
                        return interpolate.variableAnchorOffsetCollection(outputLower, outputUpper, t);
                    default:
                        return [outputLower, outputUpper, t];  // <-- New one
                }
            case 'interpolate-hcl':
                return interpolate.color(outputLower, outputUpper, t, 'hcl');
            case 'interpolate-lab':
                return interpolate.color(outputLower, outputUpper, t, 'lab');
        }
    }
```